libs/SalesforceSDKCore/SalesforceSDKCore/Classes/RestAPI/SFRestAPI.m

```diff
@@ -72,7 +72,7 @@ - (id)initWithUser:(SFUserAccount *)user {
     self = [super init];
     if (self) {
         self.user = user;
-        _activeRequests = [[NSMutableSet alloc] initWithCapacity:10];
+        _activeRequests = [SFSDKSafeMutableSet setWithCapacity:10];
```

Comment:
Shouldn't we use an ordered set instead?

Reply:
@wmathurin  Should we change the SFRestAPI to use an ordered set given the prior implementation did not care about the order?

Reply:
I see it has been a set for years. I guess if app needs the requests to go out in a deterministic order, it would wait on the response of a request before firing the next one.